### PR TITLE
:doc: Typo

### PR DESCRIPTION
**Pull request by prometherion**
Just a simple typo

### CODE REVIEW

**Comment by prometherion** [comment]
I signed it

**Comment by prometherion** [comment]
I signed it

File: docs/book/basics/simple_resource.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ field, which may be set by autoscalers.
 ```go
 // ContainerSetSpec defines the desired state of ContainerSet
 type ContainerSetSpec struct {
-  // replics is the number of replicas to maintain
+  // replicas is the number of replicas to maintain
```
Comment (mengqiy) [inline docs/book/basics/simple_resource.md]: It should match the field name, i.e. Replicas

Comment (prometherion) [reply docs/book/basics/simple_resource.md]: I think doc is referencing to JSON representation, so lowercase: just take a look:

```
  // **image** is the container image to run.  Image must have a tag.
  // +kubebuilder:validation:Pattern=.+:.+
  Image string `json:"**image**,omitempty"
```

**Pull request merged**